packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -37,6 +37,7 @@
   "force_project_create_text": "SFDX: Create Project",
   "force_project_with_manifest_create_text": "SFDX: Create Project with Manifest",
   "force_apex_trigger_create_text": "SFDX: Create Apex Trigger",
+  "force_push_on_save_description": "Specifies whether to always use --forceoverwrite when doing force:source:push on save.",
```

Comment:
@sbudhirajadoc

Reply:
Slight modification -- "Specifies whether to always use --forceoverwrite when you run force:source:push on save"